src/patternfly/components/Tabs/tabs.scss

```diff
@@ -153,14 +191,20 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
     .pf-c-tabs__item.pf-m-current {
       --pf-c-tabs__link--BackgroundColor: var(--pf-c-tabs__item--m-current__link--Background);
```

Comment:
should `--pf-c-tabs__item--m-current__link--Background` be `--pf-c-tabs__item--m-current__link--BackgroundColor`?